### PR TITLE
UI Tweaks

### PR DESCRIPTION
**Pull request by RonnyO**
Hey Liron!
That's a cool little app 👍 

Here are some tweaks, a bit opinionated, all UI-related. Cheers!
Oh, and we're hiring talented FEDs 🚀

### CODE REVIEW

**Comment by LironHazan** [comment]
Thanks Ronny!! That’s really kind of you :) one of my initial purposes was to play a bit with firestore so I didn’t put much effort on the UI but I’m really glad to get such feedback! And I’ll definitely go over the PR soon :)

File: src/app/tasks-board/new-task/new-task.component.html [hunk 1]
```diff
@@ -1,12 +1,12 @@
-<div class="new-task">
-  <input [(ngModel)]="name" type="text" placeholder="Enter title">
+<form class="new-task" name="new-task" (submit)="onSave(); $event.preventDefault()">
```
Comment (LironHazan) [inline src/app/tasks-board/new-task/new-task.component.html]: awesome! not sure why I hadn't use form at first place :) will add validations as well..

File: src/app/tasks-board/tasks-board.component.scss [hunk 2]
```diff
@@ -1,38 +1,40 @@
...
 .container:not(:last-child) {
-  margin-right: 8px;
+  margin-right: var(--tasks-h-margin);
 }
 
 .task {
+  box-sizing: border-box; // prevents the mirrored copy from changing dimensions
+  margin-top: 14px;
+  padding: 8px;
+  word-wrap: break-word;
```
Comment (LironHazan) [inline src/app/tasks-board/tasks-board.component.scss]: great!!

File: src/styles.css [hunk 3]
```diff
@@ -1,7 +1,13 @@
 /* You can add global styles to this file, and also import other style files */
 @import "../node_modules/@angular/material/prebuilt-themes/indigo-pink.css";
 
-html, body { height: 100%;
+:root {
```
Comment (LironHazan) [inline src/styles.css]: that's nice I wasn't familiar with it

**Comment by RonnyO** [comment]
Thank you Liron, had a nice time playing with this 👍

**Pull request merged**